### PR TITLE
Convert to Markdown

### PR DESCRIPTION
**Pull request by TheLastProject**
README converted to Markdown to make it show up more prettily on Github.

### CODE REVIEW

**Comment by qreeves** [comment]
I'm not sure it is a great idea to rename the file from .txt to .md. It has been this way for a long time so Windows users can double click the file to open it in their text editor. I know that sound silly, but I think it is important.

File: readme.md [hunk 1]
```diff
@@ -77,12 +77,12 @@ The project is Free and Open Source meaning that you can both use it for free, a
 * Rob "Lloir" Shannon
 * Robert "Homicidal" Crane
 * Robert Winkler
-* Ruben "TheLastProject" van Os - Design, Testing and Support
+* "TheLastProject" - Design, Testing and Support
 * Ryan "icculus" Gordon / icculus.org - Master server and website hosting
-* R�mi "LuckystrikeRx" Clouet d'Orval - Maps, Textures, Models, Weapons, Design/Testing, Website
+* Rémi "LuckystrikeRx" Clouet d'Orval - Maps, Textures, Models, Weapons, Design/Testing, Website
```
Comment (qreeves) [inline readme.md]: Also, why did this change? :stuck_out_tongue:

Comment (TheLastProject) [inline readme.md]: Github's editor changed, as it is displayed in vim for me, <E9> to é, so probably a different character encoding or so. There's no other change there.

**Comment by TheLastProject** [comment]
Valid point. I have restored readme.txt. This does mean both readme.md and readme.txt will have to be updated in case anything is to be changed, but seeing how the README is unlikely to be updated often anyway this doesn't seem like a serious problem to me.

**Comment by ienorand** [comment]
It is apparently possible to use a symlink (README.md -> readme.txt) in order to force parsing of any file on GitHub (symlinks in Git ends up as dummy files on windows, so won't cause issues).

**Comment by qreeves** [comment]
OK, yeah. Do what @ienorand said. Doesn't matter if the txt is in md format.

**Pull request closed without merging**